ogre2/src/Ogre2SelectionBuffer.cc

```diff
@@ -63,10 +64,13 @@ class ignition::rendering::Ogre2SelectionBufferPrivate
   public: Ogre::CompositorWorkspace *ogreCompositorWorkspace = nullptr;
 
   /// \brief Render texture data buffer
-  public: uint8_t *buffer = nullptr;
+  public: float *buffer = nullptr;
 
   /// \brief Ogre pixel box that contains description of the data buffer
   public: Ogre::PixelBox *pixelBox = nullptr;
+
+  /// \brief The selection buffer material
+  public: Ogre::MaterialPtr selectionMaterial;
```

Comment:
```suggestion
  public: Ogre::MaterialPtr selectionMaterial{nullptr};
```

Reply:
looks like `Ogre::SharedPtr` can't be set or init to `nullptr`.